hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/container/ContainerTestHelper.java

```diff
@@ -490,7 +527,23 @@ public static ContainerCommandRequestProto getDeleteBlockRequest(
     request.setContainerID(blockID.getContainerID());
     request.setDeleteBlock(delRequest);
     request.setDatanodeUuid(pipeline.getFirstNode().getUuidString());
-    return request.build();
+    return request;
+  }
+
+  public static Builder newGetCommittedBlockLengthBuilder(Pipeline pipeline,
```

Comment:
Similar like above, can we add a token parameter for the new methods?

Reply:
These return a builder, which the caller can further change.  The goal is to avoid proliferation of methods (with/without token, etc.).  I haven't converted all existing methods to return builder only to minimize the patch.  Can do it in a follow-up refactor.

Reply:
That makes sense. We can do it in follow up ticket.

Reply:
Thanks.  Created https://issues.apache.org/jira/browse/HDDS-5251 for this.